src/script/view_model/AuthViewModel.js

```diff
@@ -80,7 +81,7 @@ class AuthViewModel {
       this.cryptography_service,
       this.storage_repository
     );
-    this.client_service = new z.client.ClientService(backendClient, this.storageService);
+    this.client_service = new ClientService(backendClient, this.storageService);
```

Comment:
This line can be removed, you need to change the parameter of the `clientRepo` on the next line.